docs/README.hbs

```diff
@@ -40,6 +40,20 @@ Run the test suite by doing:
 $ npm test
 ```
 
+Troubleshooting
+---------------
+
+### `error C2373: '__pfnDliNotifyHook2': redefinition`
+
+This error indicates that you're running a too old npm version. Upgrade by running:
```

Comment:
I think "...that the version of npm you're running is too old." might be more readable?